Dockerfile

```diff
@@ -30,18 +31,18 @@ COPY --parents .yarn/ yarn.lock .yarnrc.yml **/package.json packages/bind-mount/
 #
 # If the following issue is ever addressed, we can use that instead:
 # https://github.com/yarnpkg/berry/issues/6339
-RUN cd /PrairieLearn && yarn dlx node-gyp install && yarn install --immutable --inline-builds && yarn cache clean
+RUN yarn dlx node-gyp install && yarn install --immutable --inline-builds && yarn cache clean
 
 # NOTE: Modify .dockerignore to allowlist files/directories to copy.
-COPY . /PrairieLearn/
+COPY . .
 
 # set up PrairieLearn and run migrations to initialize the DB
+# hadolint ignore=SC3009
```

Comment:
Should we disable this globally? I think we don't generally use plain `/bin/sh`.

Reply:
This ignore is only present in this file -- unless we can enforce not using plain `/bin/sh` I would prefer to keep this.

Reply:
Sure 👍